linkerd/protocol/http/src/test/scala/io/buoyant/linkerd/protocol/http/HttpConfigTest.scala

```diff
@@ -46,6 +49,76 @@ class HttpConfigTest extends FunSuite with Awaits {
 
   }
 
+  test("DefaultsTest - no streaming, no streamAfterContentLengthKB") {
+    val yaml = s"""
+                  |protocol: http
+                  |identifier:
+                  |  kind: io.l5d.methodAndHost
+                  |maxInitialLineKB: 4
+                  |servers:
+                  |- port: 5000
+      """.stripMargin
+    val config = parse(yaml)
+
+    val streaming = config.routerParams(Stack.Params.empty)[Streaming]
+    assert(streaming.enabled)
+    /*  wish we could assert this, cause the "default" was 5meg
+    streaming match {
+      case Streaming.Enabled(fixedLengthStreamedAfter) => assert(fixedLengthStreamedAfter == 5.kilobytes)
+      case _ => fail()
+    }
+    */
+  }
+
+  test("DefaultsTest - no streaming, streamAfterContentLengthKB present") {
+    val yaml = s"""
+                  |protocol: http
+                  |identifier:
+                  |  kind: io.l5d.methodAndHost
+                  |maxInitialLineKB: 4
+                  |streamAfterContentLengthKB: 5
+                  |servers:
+                  |- port: 5000
+      """.stripMargin
+    val config = parse(yaml)
+
+    val streaming = config.routerParams(Stack.Params.empty)[Streaming]
+    assert(streaming.enabled)
+  }
+
+  test("DefaultsTest - Stream enabled, no streamAfter") {
+    val yaml = s"""
+                  |protocol: http
+                  |identifier:
+                  |  kind: io.l5d.methodAndHost
+                  |maxInitialLineKB: 4
+                  |streamAfterContentLengthKB: 5
+                  |streamingEnabled: true
+                  |servers:
+                  |- port: 5000
+      """.stripMargin
+    val config = parse(yaml)
+
+    val streaming = config.routerParams(Stack.Params.empty)[Streaming]
+    assert(streaming.disabled)
+  }
+
+  test("DefaultsTest - Stream disabled, streamAfter Set") {
+    val yaml = s"""
+                  |protocol: http
+                  |identifier:
+                  |  kind: io.l5d.methodAndHost
+                  |maxInitialLineKB: 4
+                  |streamingEnabled: false
+                  |servers:
+                  |- port: 5000
+      """.stripMargin
+    val config = parse(yaml)
+
+    val streaming = config.routerParams(Stack.Params.empty)[Streaming]
+    assert(streaming.enabled)
```

Comment:
Same here, according to your logic, this will not disable streaming as it will hit the `(_, Some(streamAfter)) ` case

Reply:
stupid brain fart commit. Fixed.